jest.conf/index.js

```diff
@@ -10,6 +10,7 @@ const moduleNameMapper = {
 module.exports = {
   rootDir: path.resolve(__dirname, '..'),
   moduleFileExtensions: ['js', 'json', 'vue'],
+  testNamePattern: '^(?!.*\\[Visual\\])',
```

Comment:
Wooo, thats great, I researched if we had something like this for jest.config but I found nothing in the docs, but its great that this is working :tada: